### ISSUE TITLE
Exception raised

### ISSUE THREAD

**Issue by danieldw69ios** [KillianLucas/open-interpreter#241]
**Describe the bug**
Exception raised frozen runpy

**To Reproduce**
1. installed latest version of anaconda
2. conda create -n oi python=3.11.4
3. conda activate oi
4. pip install open-interpreter
5. interpreter -y
6. See error

**Expected behavior**

**Screenshots**
$ interpreter -y
> what files are on my desktop
NoneType: None
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\User\.conda\envs\oi\Scripts\interpreter.exe\__main__.py", line 7, in <module>
  File "C:\Users\User\AppData\Roaming\Python\Python311\site-packages\interpreter\interpreter.py", line 131, in cli
    cli(self)
  File "C:\Users\User\AppData\Roaming\Python\Python311\site-packages\interpreter\cli.py", line 207, in cli
    interpreter.chat()
  File "C:\Users\User\AppData\Roaming\Python\Python311\site-packages\interpreter\interpreter.py", line 287, in chat
    self.respond()
  File "C:\Users\User\AppData\Roaming\Python\Python311\site-packages\interpreter\interpreter.py", line 509, in respond
    raise Exception("")
Exception

**System (please complete the following information):**
 - OS: both on Windows and iOS
 - Python 3.11.4
 - Open Interpreter version 0.1.2

**Additional context**

**Comment by NachoVazquez** [comment]
I'm having the same issue on MacOS.

**Comment by NachoVazquez** [comment]
> I'm having the same issue on MacOS.
> 
> MacOS: 13.5.2 Python: 3.11.5 Open interpreter: 0.1.2

Uninstalling a reinstalling fixed it for me.

**Comment by kecbigmt** [comment]
I have the same issue.
Reinstalling didn't fix it for me.

- MacOS: 13.5.1
- Python: 3.10.13
- Open interpreter: 0.1.2

```
Press CTRL-C to exit.

> hi
NoneType: None
Traceback (most recent call last):
  File "/open-interpreter-workspace/.venv/bin/interpreter", line 8, in <module>
    sys.exit(cli())
  File "/open-interpreter-workspace/.venv/lib/python3.10/site-packages/interpreter/interpreter.py", line 131, in cli
    cli(self)
  File "/open-interpreter-workspace/.venv/lib/python3.10/site-packages/interpreter/cli.py", line 207, in cli
    interpreter.chat()
  File "/open-interpreter-workspace/.venv/lib/python3.10/site-packages/interpreter/interpreter.py", line 287, in chat
    self.respond()
  File "/open-interpreter-workspace/.venv/lib/python3.10/site-packages/interpreter/interpreter.py", line 509, in respond
    raise Exception("")
Exception
```

**Comment by Videoteq** [comment]
Same issue with Linux Mint (and Mac)
Python 3.11.4
Tried re-installing but no joy

Linux response:

> what files are on the desktop
NoneType: None
Traceback (most recent call last):
  File "/home/bob/anaconda3/envs/i/bin/interpreter", line 8, in <module>
    sys.exit(cli())
             ^^^^^
  File "/home/bob/anaconda3/envs/i/lib/python3.11/site-packages/interpreter/interpreter.py", line 131, in cli
    cli(self)
  File "/home/bob/anaconda3/envs/i/lib/python3.11/site-packages/interpreter/cli.py", line 207, in cli
    interpreter.chat()
  File "/home/bob/anaconda3/envs/i/lib/python3.11/site-packages/interpreter/interpreter.py", line 287, in chat
    self.respond()
  File "/home/bob/anaconda3/envs/i/lib/python3.11/site-packages/interpreter/interpreter.py", line 509, in respond
    raise Exception("")
Exception

**Comment by bakertn** [comment]
Same issue.  Using Anaconda/Windows, updated both versions of Anaconda and Python just before installing this.

Chat:  Hello

Response:


Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\lbake\anaconda3\envs\oi\Scripts\interpreter.exe\__main__.py", line 7, in <module>
  File "C:\Users\lbake\anaconda3\envs\oi\Lib\site-packages\interpreter\interpreter.py", line 131, in cli
    cli(self)
  File "C:\Users\lbake\anaconda3\envs\oi\Lib\site-packages\interpreter\cli.py", line 207, in cli
    interpreter.chat()
  File "C:\Users\lbake\anaconda3\envs\oi\Lib\site-packages\interpreter\interpreter.py", line 287, in chat
    self.respond()
  File "C:\Users\lbake\anaconda3\envs\oi\Lib\site-packages\interpreter\interpreter.py", line 509, in respond
    raise Exception("")
Exception

**Comment by bakertn** [comment]
Same issue.  Widows, Anaconda Navigator, Updated Anaconda and Python to latest versions just before installing.

Chat:  Hello

Response:
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\lbake\anaconda3\envs\oi\Scripts\interpreter.exe\__main__.py", line 7, in <module>
  File "C:\Users\lbake\anaconda3\envs\oi\Lib\site-packages\interpreter\interpreter.py", line 131, in cli
    cli(self)
  File "C:\Users\lbake\anaconda3\envs\oi\Lib\site-packages\interpreter\cli.py", line 207, in cli
    interpreter.chat()
  File "C:\Users\lbake\anaconda3\envs\oi\Lib\site-packages\interpreter\interpreter.py", line 287, in chat
    self.respond()
  File "C:\Users\lbake\anaconda3\envs\oi\Lib\site-packages\interpreter\interpreter.py", line 509, in respond
    raise Exception("")
Exception

**Comment by victoriano** [comment]
same issue here. At the beginning, it worked now it doesn't :(

**Comment by Videoteq** [comment]
It doesn't work in Colab either! There must be a fundamental flaw, but way beyond my paygrade to figure out!

**Comment by Snape888** [comment]
Same issue here:
MacOS Ventura (clean install) 13.5.2

**Comment by thenetguy** [comment]
Same issue here

**Comment by contang0** [comment]
same...

**Comment by diatche** [comment]
Judging by the solutions in #247, if this happens immediately, then the OpenAI key hasn't been exported beforehand, but if it happens later on, it could be an error while calling the OpenAI endpoint causing the process to fail. If the latter is the case, then it seems error recovery would need to be looked at.

**Comment by kecbigmt** [comment]
I got a workaround.
I tried to set OPENAI_API_KEY to environment variables before running `interpreter`.

```
export OPENAI_API_KEY=sk-xxxxxx && interpreter
```
 
And it works.

Thank you, @diatche.

**Comment by Videoteq** [comment]
Thanks @diatche 

When I opened interpreter (on my Linux machine) it promted to upgrade to v0.1.3, after which I saved the API key as you suggested and now it's all working fine (and amazing is is too!). Have to ty it on my Mac now.....

**Comment by danieldw69ios** [comment]
It worked once I upgraded to v0.1.3 using 'pip install --upgrade open-interpreter.' After that, I verified the upgrade with 'pip show open-interpreter.' Then, I executed 'set OPENAI_API_KEY=sk-xxxxx.' Afterward, I ran 'interpreter -y' and asked how many files are on my desktop? However, I encountered the following error message: 'openai.error.InvalidRequestError: The model 'gpt-4' does not exist, or you do not have access to it. Learn more: [link].' So, I then used the following command: 'interpreter --fast -y' and asked the same question, and it worked.
